### PR TITLE
refactor: Session.

### PR DESCRIPTION
**Pull request by prmukherj**

### CODE REVIEW

File: src/ansys/fluent/core/session_solver.py [hunk 1]
```diff
@@ -271,13 +271,15 @@ def _populate_settings_api_root(self):
 
     def __getattr__(self, attr):
         self._populate_settings_api_root()
+        if attr in [x for x in dir(self._settings_api_root) if not x.startswith("_")]:
+            warnings.warn(
+                f"'{attr}' is deprecated. Use 'settings.{attr}' instead.",
+                DeprecationWarning,
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.py:277]: Please use a custom class for this warning, so that it can be easily disabled if ever needed.

Comment (mkundu1) [reply src/ansys/fluent/core/session_solver.py:277]: There are currently custom warnings `DeprecatedSettingWarning` and `UnstableSettingWarning` for settings API. The formaer can be used for the currrent purpose.

**Comment by mkundu1** [comment]
Please add tests for the new warnings.

**Comment by prmukherj** [comment]
> Please add tests for the new warnings.

Will do, actually other refactoring for session level will also be done with this.

**Comment by prmukherj** [comment]
> > Please add tests for the new warnings.
> 
> Will do, actually other refactoring for session level will also be done with this.

@mkundu1, No new warnings have been added as part of this PR. We have decided to directly make these changes without adding deprecation warnings.

**Review by mkundu1 — APPROVED** [review]
Can you please verify PyConsole with this restructuring? These changes will be packaged in Fluent's PyFluent sometime during 25.1 development but we can verify that before merging the PR. You can set `FLUENT_PYFLUENT_ROOT` env var to pickup your local PyFluent in PyConsole.

**Comment by prmukherj** [comment]
> Can you please verify PyConsole with this restructuring? These changes will be packaged in Fluent's PyFluent sometime during 25.1 development but we can verify them before merging this PR. You can set `FLUENT_PYFLUENT_ROOT` env var to pickup your local PyFluent in PyConsole.

Few minor issues needs fixing for 25.1. I have them locally and checked it's working as well. I'll push them as soon as the develop branch for 25.1 is available.

**Review by hpohekar — APPROVED** [review]

**Review by ajain-work — APPROVED** [review]

**Pull request merged**